src/plugins/intel_cpu/tests/unit/streams_info/streams_e2e_test.cpp

```diff
@@ -579,6 +579,27 @@ StreamGenerateionTestCase generation_tput_2sockets_48cores_9 = {
 };
 
 #ifdef __linux__
```

Comment:
Since Windows and Linux use same test data now, please use `#if defined(__linux__) || defined(_WIN32)` instead of duplicate test data.

Reply:
Ok, updated.